contracts/user-contracts/src/main/java/org/java/mentorship/contracts/user/dto/User.java

```diff
@@ -2,12 +2,19 @@
 
 import lombok.Data;
 
+import java.time.OffsetDateTime;
+
 @Data
 public class User {
     private Integer id;
 
     private String email;
     private String firstName;
     private String lastName;
-    private Boolean verified;
+
+    private OffsetDateTime createdAt;
+    private OffsetDateTime verifiedAt;
+    private OffsetDateTime lastSentVerificationNotification;
```

Comment:
Don't think lastSentVerificationNotification and lastSentPasswordChangeToken provide any advantage in this table.
If we were to use them in nay flow, one could fetch them from the notification service.